### PR TITLE
Conversion Boost

### PR DESCRIPTION
**Pull request by adamgordonbell**
Attempt to boost the conversion rate of the blog by adding more CTAs.

Here are the CTAs:

# Generic:

Screen Shot 2023-03-28 at 4 37 00 PM
Screen Shot 2023-03-28 at 4 37 16 PM

# Docker:

Screen Shot 2023-03-28 at 4 36 35 PM
Screen Shot 2023-03-28 at 4 36 46 PM

# Make:

Screen Shot 2023-03-28 at 4 37 54 PM

Screen Shot 2023-03-28 at 4 38 05 PM

# Go:

Screen Shot 2023-03-28 at 4 42 47 PM

# More Generic (used where nothing better for in article CTA) :

Screen Shot 2023-03-28 at 4 39 19 PM
Screen Shot 2023-03-28 at 4 40 36 PM

# Python:

Pending ....

### CODE REVIEW

File: blog/_includes/cta/blog-newsletter1.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!-- Section: Design Block -->
```
Comment (adamgordonbell) [inline blog/_includes/cta/blog-newsletter1.html:1]: I pulled this styling from the tailwind examples website and customized it as needed.

**Comment by vladaionescu** [comment]
Hey - looks pretty good! Is there an easy way to track which of the panels was clicked on within an article? I'm thinking that we could use that to tune the text within the panel over time.

**Review by vladaionescu — APPROVED** [review]

**Comment by thtmnisamnstr** [comment]
This looks great. I'd actually like to see this on every Docker, Make, and Go (and eventually Python) post we have, not strictly the high-performing ones.

Also, I agree with Vlad. It would be good if we could track which ones of these are clicked. After we have that, we could also make some goals and funnels in Google Analytics for these.

**Review by thtmnisamnstr — APPROVED** [review]

**Comment by adamgordonbell** [comment]
> Also, I agree with Vlad. It would be good if we could track which ones of these are clicked. After we have that, we could also make some goals and funnels in Google Analytics for these.

I'll take a look at what we can track. It's a good idea.

**Pull request merged**